src/components/panels/TimePanel.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <b-col md="6">
+            <h3 class="text-center">{{headertext}}</h3>
```

Comment:
tabs? spaces? looks like possible mismatch

Reply:
When I click-drag with the mouse, they all look like spaces.  The highlighting with tab characters should jump.